Vector2D trueVel
Vector2D cmdVel
